app/models/course/level.rb

```diff
@@ -20,6 +20,14 @@ def default_level?
     experience_points_threshold == 0
   end
 
+  # Returns the next higher level in the course
+  # nil is returned if current level is the highest level
+  #
+  # @return [Course::Level, nil]
```

Comment:
Looks like there should be multiple `@return` tags?
http://www.rubydoc.info/gems/yard/file/docs/Tags.md#return